packages/hint/docs/contributor-guide/getting-started/events.md

```diff
@@ -224,6 +225,31 @@ type CanEvaluateScript {
 }
 ```
 
+## `can-traverse::html`
+
+Event is emitted **when** the `connector` is ready to traverse
+the html.
```

Comment:
What does it mean to be ready?

Reply:
I have change `is ready` for `wants to`, is that better?